shenyu-common/src/main/java/org/apache/shenyu/common/enums/OperatorEnum.java

```diff
@@ -84,7 +84,7 @@ public enum OperatorEnum {
     /**
      * Path patter operator enum.
      */
-    PATH_PATTER("pathPatter", true);
+    PATH_PATTERN("pathPattern", true);
```

Comment:
sql script  is pathPattern？

Reply:
yes, u can see the img[search in sql script](https://user-images.githubusercontent.com/49482363/182755359-c1bdf2e2-39df-45fe-9163-22a7bf8dcb9a.png).